packages/pds/src/repo/prepare.ts

```diff
@@ -127,6 +129,22 @@ export const assertValidRecord = (record: Record<string, unknown>) => {
   }
 }
 
+// additional more rigorous check on datetimes
+// this check will eventually be in the lex sdk, but this will stop the bleed until then
+export const assertValidCreatedAt = (record: Record<string, unknown>) => {
+  const createdAt = record['createdAt']
+  if (typeof createdAt !== 'string') {
+    return
+  }
+  try {
+    ensureValidDatetime(createdAt)
```

Comment:
added the extra check here

This means records with bad timestamps will still pass "lexicon validation" (which we want so that old records don't break application views), but they will no longer be able to be created on the pds

this specific approach only works as is because the only datetimes on our records are `createdAt` times & every `createdAt` property is a datetime. if we get other datetimes, we'll need to tweak this logic